addons/account_debit_note/wizard/account_debit_note.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, api
+from odoo.tools.translate import _
+
+
+class AccountDebitNote(models.TransientModel):
+    """
+    Add Debit Note wizard: when you want to correct an invoice with a tiny amount of something
+    """
+    _name = 'account.debit.note'
+    _description = 'Add Debit Note wizard'
+
+    move_id = fields.Many2one('account.move', string='Journal Entry',
+        domain=[('state', '=', 'posted'), ('type', 'not in', ('out_refund', 'in_refund'))])
+    date = fields.Date(string='Adjustment date', default=fields.Date.context_today, required=True)
+    reason = fields.Char(string='Reason')
+    journal_id = fields.Many2one('account.journal', string='Use Specific Journal', help='If empty, uses the journal of the journal entry to be reversed.')
+
+    # computed fields
+    residual = fields.Monetary(compute="_compute_from_moves")
+    currency_id = fields.Many2one('res.currency', compute="_compute_from_moves")
+    move_type = fields.Char(compute="_compute_from_moves")
```

Comment:
I would add a **Product ID**

A debit note is almost never with the same lines of the invoice, it is almost always products that **adjust** the original invoice.

**Interest for late payment** for example.
**Unchargable check** is other common one.

Reply:
The debit notes will always be created in draft.  I am thinking about just one checkbox "Copy Lines" which is not checked by default.